package.json

```diff
@@ -15,44 +15,42 @@
     "@microsoft/applicationinsights-react-js": "^17.3.2",
     "@microsoft/applicationinsights-web": "^3.3.3",
     "ansi-regex": "6.0.1",
-    "axios": "1.6.4",
-    "babel-traverse": "^6.26.0",
+    "axios": "1.7.4",
     "body-parser": "1.20.3",
-    "braces": "2.3.1",
+    "braces": "3.0.3",
     "constantinople": "3.1.1",
     "cookie": "0.7.0",
     "decode-uri-component": "0.2.2",
     "dexie": "^3.2.3",
     "dexie-encrypted": "^2.0.0",
     "ejs": "3.1.10",
-    "engine.io": "6.4.2",
+    "engine.io": "6.6.2",
     "es5-ext": "0.10.63",
-    "express": "4.19.2",
+    "express": "4.21.1",
     "follow-redirects": "1.15.6",
     "generic-type-guard": "^3.7.0",
     "get-func-name": "2.0.1",
-    "glob": "^9.0.0",
+    "glob": "9.0.1",
```

Comment:
Previously we only required a minimum of 9.0.0, but now we require exactly 9.0.1. Is there a reason we want to prevent versions higher than 9.0.1?